packages/web-components/src/message-bar/message-bar.template.ts

```diff
@@ -0,0 +1,25 @@
+import { ElementViewTemplate, html } from '@microsoft/fast-element';
+import type { MessageBar } from './message-bar.js';
+
+/**
+ * Generates a template for the MessageBar component.
+ * @public
+ * @param {MessageBar} T - The type of the MessageBar.
+ * @returns {ElementViewTemplate<T>} - The template for the MessageBar component.
+ */
+export function messageBarTemplate<T extends MessageBar>(): ElementViewTemplate<T> {
+  return html<T>`
+    <slot name="icon"></slot>
+    <div class="content">
+      <slot></slot>
+    </div>
+    <slot name="actions"></slot>
```

Comment:
I dropped two actions in while testing this to see if it would support it, but it looks like some of the styling is off (spacing between the action and the close button, the two actions overlap in the same position):

<img width="1265" alt="image" src="https://github.com/microsoft/fluentui/assets/13071055/46c7664d-5d16-4d45-854d-d8ed24acea1f">

Reply:
I wasn't able to figure out how to achieve the intended design without adding an additional wrapping `div` around the Actions. I didn't prescribe any spacing of the buttons within as I figured it would be best for the implementor to add this.

![image](https://github.com/microsoft/fluentui/assets/49389/40c49ad1-10b6-485c-a7a7-0aec07eba964)